### PR TITLE
[Fix] - OpenData Failing Job

### PR DESCRIPTION
**Pull request by AyakorK**
#### 🎩 Description

This PR is made to fix the OpenDataJob that was crashing if the author of a proposal was an organization because of the overriding of the serializer.

### 📌 Related Issues

Related to [Notion](https://www.notion.so/opensourcepolitics/Toulouse-OpenDataJob-fails-and-blocks-exports-queue-f926e55a82e347eba4cd0dd5f29dce89?pvs=4)

### 📝 What has been done

- Fix Serializer
- Change queue of the job
- Add some specs
- Rewrite the cron to match the format of a normal cron task
- Bump Sidekiq to v7.0.7 (latest version)

### CODE REVIEW

**Review by paulinebessoles — APPROVED** [review]

File: lib/extends/exporters/proposal_serializer.rb [hunk 1]
```diff
@@ -50,18 +49,37 @@ def serialize
...
+  def creator_email
+    return "" if proposal.creator_identity.blank?
+
+    proposal.creator_identity.try(:email).presence || ""
+  end
+
+  def creator_phone_number
+    return "" if proposal.creator_identity.blank?
+
+    phone_number(proposal.creator_identity.try(:user_id)) || ""
```
Comment (Quentinchampenois) [inline lib/extends/exporters/proposal_serializer.rb:74]:
```suggestion
    return "" if proposal.creator_identity.try(:user_id).blank?

    phone_number(proposal.creator_identity.try(:user_id))
```

File: lib/extends/exporters/proposal_serializer.rb [hunk 2]
```diff
@@ -50,18 +49,37 @@ def serialize
...
+  end
+
   # phone_number retrieve the phone number of an user stored from phone_authorization_handler
   # Param: user_id : Integer
   # Return string, empty or with the phone number
   def phone_number(user_id)
+    return "" if user_id.blank?
```
Comment (Quentinchampenois) [inline lib/extends/exporters/proposal_serializer.rb:81]: If we add condition in `create_phon_number`, this guard clause is no longer needed

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
Nice work, I just have a few changes suggestions

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
I come back on the suggestion I made to create the queue `low`,
 I will fix it today on this PR, and reduce the retry count from default (25) to 2 because it is a scheduled job

**Comment by Quentinchampenois** [comment]
I keep the `low` queue because main issue was OpenDataJob was blocking the whole queue and admins can't receive exports from application

**Review by Quentinchampenois — APPROVED** [review]
LGTM

**Pull request merged**